idaes_examples/notebooks/docs/flowsheets/hda_flowsheets_for_costing_notebook.py

```diff
@@ -52,6 +52,7 @@
 import idaes.core.util.scaling as iscale
```

Comment:
Does the HDA with costing notebook use this, and do those files need to be updated/re-run?

Reply:
Yes, the code does set the scaling for a few components with iscale. I do not think the files should be updated.